### DISCUSSION TITLE
【北京】听友线下聚会，AA 制

### DISCUSSION THREAD

**Discussion by LinuxSuRen** [opensource-f2f/episode#145] [Polls]
本次《开源面对面》听友线下聚会，由 @LinuxSuRen 发起。

大家可以参考下面的话题来交流：
* 开源面对面节目相关
* 任何开源相关
  * 吐槽
  * 有趣的经历
  * 个人体验
  * 困惑
* 其他

时间：周日，2022-07-17（预计下午两点后）

报名方法：
* 在帖子后面留言（表示要报名即可）
* 加 @LinuxSuRen 微信，进微信群

注意事项：
* 这次活动没有人数预期，没有预期达成任何目标
* 活动费用以 AA 制为主，如有赞助的话，费用会平均减少
* 活动没有演讲要求，但也欢迎有兴趣的同学来个快闪分享、交流
* 以圆桌交流为主（可能是大圆桌或小圆桌，根据具体场地以及大家意愿等情况来定）

> 如果您有合适的（距离适中、免费、可以容纳 10 人左右、夏季有空调），欢迎联系 @LinuxSuRen ，或者直接在下面添加回复。
> [这里](https://github.com/LinuxSuRen/open-source-best-practice/issues/43)是 @LinuxSuRen 上次发起的 AA 制聚会活动。

**Comment by juzhiyuan** [comment]
🤣🤣 想去但是去不了

**Reply by LinuxSuRen** [reply]
这次在北京先踩坑，大家后续可以在其他城市搞起来。

**Reply by wey-gu** [reply]
+1

**Reply by MikeyWei** [reply]
第一次听友聚会，一定要拍照留念。

**Reply by RenChauncy** [reply]
我参加

**Comment by miao-gm** [comment]
举手，在北京想去参加

**Comment by LinuxSuRen** [comment]
很开心看到各位对本次活动有兴趣，参加本次活动的朋友可以加我微信（同 GitHub ID），把您拉进微信群中，方便到时候联系。

**Comment by faweizhao26** [comment]
报名报名！

**Comment by LinuxSuRen** [comment]
很开心与各位交流！！！

![8c9992052b4f6249d3a62d9304cb797](https://user-images.githubusercontent.com/1450685/179400102-e3f5a6ca-1a50-4304-b878-cbf8250ea929.jpg)

**Comment by LinuxSuRen** [comment]
下次由 @LinuxSuRen 发起的活动 https://github.com/LinuxSuRen/open-source-best-practice/issues/100

**Comment by MikeyWei** [comment]
期待后续

**Reply by LinuxSuRen** [reply]
五一节后搞起来
